### PR TITLE
Create user table and constraints, facebook login creates an account if it doesn't already exist, profile info shows if logged in.

### PR DESCRIPTION
**Pull request by GameKyuubi**
Create user table and constraints, facebook login creates an account if it doesn't already exist, profile info shows if logged in.

### CODE REVIEW

File: migrations/201709211605_transaction.js [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
+      .index();
+    t.integer('event_id')
+      .unsigned()
+      .notNullable();
+    t.foreign('event_id')
+      .references('event.id');
+    t.integer('total')
+      .notNullable();
+    t.string('charge_id')
+      .notNullable();
```
Comment (kumiko-haraguchi) [inline migrations/201709211605_transaction.js]: @GameKyuubi Transaction table seems to refer to userId as FK. Is it missing here?

File: src/reducers/index.js [hunk 2]
```diff
@@ -98,6 +98,7 @@ const reducer = (state = initialState, action) => {
       newState = state;
       break;
   }
+  console.log('newstate', newState);
```
Comment (kumilange) [inline src/reducers/index.js]: @GameKyuubi Let's remove this :)

File: src/containers/Header.js [hunk 3]
```diff
@@ -48,7 +63,17 @@ const mapDispatchToProps = (dispatch) => {
...
+      const facebook = window.hello('facebook');
+      facebook.login(
+        {
+          scope: 'email',
+          force: true,
+        },
+      ).then(() => {
+        return facebook.api('me');
+      }).then((result) => {
+        console.log(result);
```
Comment (kumilange) [inline src/containers/Header.js]: @GameKyuubi Where does this result go after `console.log(result)`?

**Comment by kumilange** [comment]
@GameKyuubi LGTM! 👍

**Review by kumilange — DISMISSED** [review]

**Review by kumilange — APPROVED** [review]

**Pull request merged**